Assets/GothicVR/Scripts/Manager/ConstantsManager.cs

```diff
@@ -18,8 +18,11 @@ public class ConstantsManager : SingletonBehaviour<ConstantsManager>
 
         //Collection of PlayerPref entries for settings
         public string moveSpeedPlayerPref = "MoveSpeed";
+
         public string turnSettingPlayerPref = "TurnSetting";
 
+        public float moveSpeed = 8f;
+
         public string selectedWorld = "world.zen";
```

Comment:
General discussion point: Is constantsmanager the right place to store the loaded world information? Player prefabs are maybe also not right as for options. Maybe it's the same place where player and world parameters are saved eg a json.